### PR TITLE
Quick fix

### PR DESCRIPTION
**Pull request by TitouanGragnic**
rename Preset/doppler_8b_384_28.json -> Preset/doppler_8b_384_27.json 
and rename nb_image_buffer_ var to buffer_part_count_

### CODE REVIEW

**Review by Baschtien — COMMENTED** [review]
No changelog update, is it needed ?

File: Preset/doppler_8b_384_27.json [hunk 1]
```diff
@@ -87,7 +87,7 @@
      }
     },
     "image_rendering": {
-     "batch_size": 16,
+     "batch_size": 32,
```
Comment (Hpn4) [inline Preset/doppler_8b_384_27.json:90]: Why changing the batch size ?

Comment (TitouanGragnic) [reply Preset/doppler_8b_384_27.json:90]: because of Mickael

Comment (Baschtien) [reply Preset/doppler_8b_384_27.json:90]: i think Michael asked for it

**Comment by Hpn4** [comment]
In `phantom.ini` there is a `NbImagesPerBuffer = 8`

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,9 +3,11 @@
 ### 13.6.0
 
 - Support for Alvium Camera
-- Changed default preset to 'doppler_8b_384_28'
+- Changed default preset to 'doppler_8b_384_27'
+- Rename preset 'doppler_8b_384_28' to 'doppler_8b_384_27'
 - Default display rate is now 24
 - Added an executable desktop shortcut for ProcessHoloFiles.ps1
+- Rename ini config variable 'NbImageBuffer' to 'BufferCountPart' for camera phantom_s710 / phantom_s711 / phantom_s991
```
Comment (Hpn4) [inline CHANGELOG.md:10]: Also in Euresys

Comment (TitouanGragnic) [reply CHANGELOG.md:10]: there is no euresys ini file, it's not a camera it's for the frame grabber, user dont need to know it

**Review by Hpn4 — APPROVED** [review]

**Pull request merged**